### ISSUE TITLE
[Bug]: read from next item restarts at top of page

### ISSUE THREAD

**Issue by hexylena** [mozilla-mobile/fenix#22433]
### Steps to reproduce

1. Visit a page with a lot of text which you would like to read out, e.g. https://gutenberg.org/cache/epub/1524/pg1524-images.html
2. Talkback is on
3. Find a location from which you want to start reading (maybe where you last were in the story). Use paragraph level navigation to move around and see that it is moving backwards and forwards just fine within the page
4. Use gestures to access Android talkback menu and use "read from next item"

### Expected behaviour

Firefox should read from the next item on the page onwards until there are no items left

### Actual behaviour

Instead the page scrolls to the top and the very first accessible control is read outout, losing your place in the document

### Device name

Google Pixel 4a

### Android version

Android 12

### Firefox release type

Firefox

### Firefox version

94.1.2 (Build #2015844587) AC: 94.0.12, e5eb726f81 GV: 94.0-20211028161635 AS: 85.4.0

### Additional information

TalkBack version 12.1

I use TalkBack ok and off, I don't need it every day, so I cannot pinpoint when this happened precisely, however I would say 2-4 months ago this was working fine, the next items were read once that menu option was used.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-18626)

**Comment by DiegoARL38** [comment]
I believe that this started with TalkBack version 12.1. There are a lot of instances where this behaviour applies, not just in Firefox, or this TalkBack function.

I've got a Redmi Note 8 with Android 11.

**Comment by hexylena** [comment]
Interesting @DiegoARL38! I didn't experience this bug in chrome or other apps e.g. guardian, both read from next item properly.

**Comment by DiegoARL38** [comment]
It happens in WhatsApp chats, Twitter threads, and reading emails in Outlook, in my daily use. It's not just for Read from next item, other options from the TalkBack context menu seem to lose where the focus was before the context menu was opened..

Taking into account some apps have this problem, I would assume there are ways to mitigate or fix the issue, or TalkBack changed its internal logic and broke something between 9.1 and 12.1.

**Comment by DiegoARL38** [comment]
By the way, @hexylena , you can assign a gesture to Read from next item to circumvent the issue. That's what I did, since the problem appears to be related to the TalkBack context menu.

**Comment by hexylena** [comment]
@DiegoARL38 that makes sense, I wondered if it wasn't something like that that internal google apps (for example) were working around. 

That's a great alternative! I'll assign a gesture since it's such a common action for me anyway. Thank you :)
